src/main/java/me/botsko/prism/listeners/PrismEntityEvents.java

```diff
@@ -298,8 +298,11 @@ public void onEntityDeath(final EntityDeathEvent event) {
             if (Prism.getIgnore().event("item-drop", p)) {
                 if (!event.getDrops().isEmpty()) {
                     for (final ItemStack i : event.getDrops()) {
-                        RecordingQueue.addToQueue(ActionFactory.createItemStack("item-drop", i, i.getAmount(), -1, null,
-                                p.getLocation(), p));
+                        if (i != null) {
+                            RecordingQueue.addToQueue(ActionFactory.createItemStack("item-drop", i,
+                                i.getAmount(), -1, null, p.getLocation(), p
```

Comment:
Codacy found an issue: ['i' has incorrect indentation level 32, expected level should be 34.](https://app.codacy.com/gh/AddstarMC/Prism-Bukkit/pullRequest?prid=6282457)